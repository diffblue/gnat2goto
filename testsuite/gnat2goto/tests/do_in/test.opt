ALL XFAIL cbmc does not fail a range check at line 19 when E = 1
