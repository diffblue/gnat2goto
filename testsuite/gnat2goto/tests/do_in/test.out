[dec.assertion.1] line 16 assertion E in Small_Int: SUCCESS
[do_in_int.assertion.1] line 29 assertion I in Small_Int: SUCCESS
[do_in_int.assertion.2] line 35 assertion I in Small_Int: SUCCESS
[do_in_int.assertion.3] line 40 assertion I in Small_Int: SUCCESS
[do_in_int.assertion.4] line 44 assertion I in Small_Int: SUCCESS
[do_in_int.assertion.5] line 47 assertion I = 3: SUCCESS
[inc.assertion.2] line 7 assertion E in Small_Int: SUCCESS
[inc.assertion.1] line 10 Ada Check assertion: FAILURE
VERIFICATION FAILED
